tests/helpers/graphile.ts

```diff
@@ -4,7 +4,7 @@ import { Pool } from 'pg'
 import { PluginsServerConfig } from '../../src/types'
 
 export async function resetGraphileSchema(serverConfig: PluginsServerConfig): Promise<void> {
-    const graphileUrl = serverConfig.JOB_QUEUE_GRAPHILE_URL || serverConfig.DATABASE_URL
+    const graphileUrl = serverConfig.JOB_QUEUE_GRAPHILE_URL || serverConfig.DATABASE_URL || undefined
```

Comment:
Can't be `serverConfig`, since it's directly going into the pool, how about I use the default it was supposed to be for tests? (since this is in tests?)

Reply:
Err, I meant using the non-null assertion operator

Reply:
Ah, right, I missed that this is a helper for tests. In that case yep, good to just assert non-null